_search-plugins/search-relevance/optimize-hybrid-search.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: Optimizing hybrid search
+nav_order: 60
+parent: Using Search Relevance Workbench
+grand_parent: Search relevance
+has_children: false
+has_toc: false
+---
+
+# Optimizing hybrid search
+
+When you use hybrid search in OpenSearch, a key question is how to combine results from lexical and vector-based search effectively. OpenSearch provides different techniques with various parameters you can experiment with to find the best setup for your application. What works best, however, depends heavily on your data, user behavior, and application domain—there is no one-size-fits-all solution.
+
+The Search Relevance Workbench helps you systematically find the ideal set of parameters for your needs.
+
+## Requirements
+
+Internally, optimizing hybrid search involves running multiple search quality evaluation experiments. For these experiments, you need a query set, judgments, and a search configuration.
+Currently, the Search Relevance Workbench supports hybrid search optimization with up to two query clauses. While hybrid search typically combines vector and lexical queries, you can run hybrid search optimization with two lexical query clauses:
```

Comment:
There must be exactly two sub-queries in hybrid query, in case it's only one sub-query there will be a validation error.